lib/runnable.js

```diff
@@ -254,7 +251,7 @@ Runnable.prototype.resetTimeout = function () {
  * @private
  * @param {string[]} globals
  */
-Runnable.prototype.globals = function (globals) {
+globals(globals) {
```

Comment:
Let's take this opportunity to un-shadow this variable by renaming it

```suggestion
globals(globalArgs) {
```